.github/workflows/staging-docker-build-push.yml

```diff
@@ -26,7 +27,7 @@ jobs:
         uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
 
       - name: Build Docker image
-        run: docker build -t forms/api .
+        run: docker build -t forms/api . --build-arg ENVIRONMENT_MODE=$ENVIRONMENT_MODE
```

Comment:
Could this just be passed in as an env var at runtime using the ECS task definition?

Reply:
I guess so. I was wondering what is the benefit of having it here or in the ECS task when this is not a secret information.

Reply:
It appears to also be available at runtime with this solution

Reply:
The benefit is that the Docker image can just remain more generic.

Reply:
Got it! I will create an infra pull request to add it :)

Reply:
@patheard Here it is https://github.com/cds-snc/forms-terraform/pull/797